src/Adapters/EventSource.Netstandard13/Implementation/EventDataExtensions.cs

```diff
@@ -32,6 +32,9 @@ public static class EventDataExtensions
             SeverityLevel.Verbose       // EventLevel.Verbose == 5
         };
 
+        private static readonly string ProviderNameProperty = "ProviderName";

```

Comment:
Consider `const` instead of `static readonly` for these two strings.

Reply:
Yup, good shout.